test/spec/services/membership/membershipSpec.js

```diff
@@ -169,19 +186,55 @@ describe('MembershipService', function() {
   });
 
   describe('#getSubjectKinds', function() {
-    // No test needed, this returns static data
+    it('should return the kind list in the preferred ourder', function() {
+      var kinds = MembershipService.getSubjectKinds();
```

Comment:
Oop, my test is actually wrong, this function isn't supposed to care about sort order.  The 'mapRolebindingsForUI' handles that.